src/main/java/seedu/tp/commands/GroupCommand.java

```diff
@@ -18,6 +18,8 @@
      * @param groupList    groupList where the groupCommand execute on
      */
     public GroupCommand(GroupFactory groupFactory, GroupList groupList) {
+        assert groupList != null : "Invalid null GroupList!";
```

Comment:
Consider adding another assertion for `GroupFactory `?

Reply:
Noted and updated, removed logs from the commit too, thanks!

Reply:
Hmm this is really strange, at first the code passes checkstyle but after I commit again, it then shows me the checkstyle failed message due to missing Javadoc comments. The checkstyle test seems to be unreliable.

Reply:
> Hmm this is really strange, at first the code passes checkstyle but after I commit again, it then shows me the checkstyle failed message due to missing Javadoc comments. The checkstyle test seems to be unreliable.

Yeah I noticed this also, I guess when the method is very short (just a few lines), it does not demand a JavaDoc comment, but when it gets longer a JavaDoc comment is required